### PR TITLE
Update guide-for-system-integrators.md

### PR DESCRIPTION
**Pull request by davidiw**
### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2492)

### CODE REVIEW

File: developer-docs-site/docs/guides/guide-for-system-integrators.md [hunk 1]
```diff
@@ -1,43 +1,44 @@
 ---
-title: "Guide for System Integrators"
+title: "System Integrators' Guide"
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/guides/guide-for-system-integrators.md:2]: I am always reluctant to put apostrophes in the title. Just doesn't feel right. I suggest simply say "System Integrators Guide" without the apostrophe.

File: developer-docs-site/docs/guides/guide-for-system-integrators.md [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
 
-# Guide for System Integrators
-
-If you provide blockchain services to your customers and wish to add the Aptos blockchain to your platform, then this guide is for you. 
-
-This system integrators guide will walk you through all you need to integrate the Aptos blockchain into your platform. After integrating the Aptos blockchain, you can create accounts, create and submit and view the transactions, deploy and operator validator nodes and participate in staking in the Aptos blockchain.
-
 :::tip 
-This is an ongoing documentation. We will add more sections on a daily basis.
+This is documentation is currently under construction with more being added on a regular basis.
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/guides/guide-for-system-integrators.md:7]: This documentation

File: developer-docs-site/docs/guides/guide-for-system-integrators.md [hunk 3]
```diff
@@ -1,43 +1,44 @@
...
 :::
 
-## Prerequisites
+# System Integrators' Guide
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/guides/guide-for-system-integrators.md:10]: Suggest remove the apostrophe. These days writing "System Integrators Guide" is perfectly fine English.

File: developer-docs-site/docs/guides/guide-for-system-integrators.md [hunk 4]
```diff
@@ -1,43 +1,44 @@
...
 
-This guide assumes that you are familiar with the blockchains in general,  and in specific the command-line interfaces and other concepts discussed throughout this Aptos developer documentation site. 
+If you provide blockchain services to your customers and wish to add the Aptos blockchain to your platform, then this guide is for you. This system integrators guide will walk you through all you need to integrate the Aptos blockchain into your platform. This guide assumes that you are familiar with the blockchains.
 
 ## Overview
 
-As a system integrator you will perform some or all of these below tasks on behalf of your users: 
-
+This guide will overview the following core concepts for integrating with Aptos:
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/guides/guide-for-system-integrators.md:16]: will provide an overview of the following core concepts...

File: developer-docs-site/docs/guides/guide-for-system-integrators.md [hunk 5]
```diff
@@ -1,43 +1,44 @@
...
 
-An [account](https://aptos.dev/concepts/basics-accounts) represents a resource on the Aptos blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources. On Aptos, accounts must be created on-chain prior to any operations involving that account. The Aptos framework supports implicitly creating accounts when transferring Aptos coin via `account::transfer`.
+An [account](https://aptos.dev/concepts/basics-accounts) represents a resource on the Aptos blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources. On Aptos, accounts must be created on-chain prior to any blockchain operations involving that account. The Aptos framework supports implicitly creating accounts when transferring Aptos coin via `account::transfer` or explicitly via `account::create_account`.
+
+At creation, an [Aptos account]() contains:
+* A [resource containing Aptos Coin]() and deposit and withdrawal of coins from that resource.
+* An authentication key associated with their current public, private key(s).
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/guides/guide-for-system-integrators.md:31]: Curious, are the empty `()` meant to include linked URLs later?

File: developer-docs-site/docs/guides/guide-for-system-integrators.md (hunk 2 above)
Comment (davidiw) [reply developer-docs-site/docs/guides/guide-for-system-integrators.md:7]: HAHAHAAH no, I prefer THIS IS

File: developer-docs-site/docs/guides/guide-for-system-integrators.md (hunk 5 above)
Comment (davidiw) [reply developer-docs-site/docs/guides/guide-for-system-integrators.md:31]: oops yes distracted

**Pull request merged**